lib/makeSequence.m

```diff
@@ -191,7 +191,7 @@
             end
             
             currF0 = cfg.F0s(currF0idx); 
-            currAmp = cfg.F0sAmp(currF0idx);
+            currAmp = cfg.F0sAmpGain(currF0idx) * cfg.baseAmp;
```

Comment:
I see that `makeStim` function doesn't have `baseAmp` anymore so it make sense to have it here. May I ask why did you prefer this way?